docs/reference/method/MongoDBCollection-explain.txt

```diff
@@ -29,7 +29,7 @@ Definition
 Parameters
 ----------
 
-``$explainable`` : :phpclass:`MongoDB\\Operation\\Explainable`
```

Comment:
Note: `\\Explainable` was missed in the original v1.16 PR: https://github.com/mongodb/mongo-php-library/pull/1239#discussion_r1511407703